docs/css-style-guide.md

```diff
@@ -0,0 +1,92 @@
+Hypothesis CSS Style Guide
+==========================
+
+Hypothesis' CSS generally follows the principles of [Trello's CSS Style Guide](https://github.com/trello/trellisheets)
```

Comment:
I wouldn't start with a link that sends people off to a different CSS styleguide. By all means acknowledge the heritage at the end, but our styleguide should be (largely) self-contained, and not require or suggest that people read other folks' styleguides -- which, after all, will very likely change over time.